### PR TITLE
Added Twitch reconnect messages

### PR DESCRIPTION
**Pull request by MattEqualsCoder**

### CODE REVIEW

File: Profiles/Sassy/responses.yml [hunk 1]
```diff
@@ -775,8 +775,18 @@ Chat:
...
+  - Text: I was really hoping for a longer break from chat, but I'm now reconnected.
+  - Text: I'm reconnected to Twitch chat. And chat, don't even try saying Hey Tracker. Don't think I've forgotten who already said it.
+    Weight: 0.1
+  - Text: Reconnected to Twitch chat. Bet you were worried about the G T big key guessing game, huh?
+    Weight: 0.1
   WhenDisconnected:
-  - Text: Error with Twitch chat connection. Please save and restart tracker.
+  - Text: Error with Twitch chat connection. Good, I'm happy to get a quick break from chat. I'm sure I'll be back soon, though.
+  - Text: Looks like Twitch had an oopsy and I lost connection. I'll try to reconnect soon.
+    Weight: 1000
```
Comment (Vivelin) [inline Profiles/Sassy/responses.yml:788]: Caught one.

Comment (MattEqualsCoder) [reply Profiles/Sassy/responses.yml]: Good catch. For some reason, I thought part of the tests was checking for these high weight values. Maybe that was before it was moved to the separate repo.

**Review by CPColin — APPROVED** [review]

File: Profiles/Sassy/responses.yml (hunk 1 above)
Comment (Vivelin) [reply Profiles/Sassy/responses.yml]: I think that was a unit test in the C# code?

Comment (MattEqualsCoder) [reply Profiles/Sassy/responses.yml]: Yeah, probably so. Could possibly add a simple reflex check in this repo for a line that's "Weight:" with at least 3 numbers. I can't imagine we'd need a weight of 100+

Comment (Vivelin) [reply Profiles/Sassy/responses.yml]: Just be careful of how you do that because we do have weights of like, 0.0005.

**Pull request merged**